parts/linux/cloud-init/artifacts/ensure-no-dup.sh

```diff
@@ -35,8 +35,8 @@ if [[ -z "${bridgeIP}" ]]; then
     exit 1
 fi
 
-podSubnetAddr=$(cat /etc/cni/net.d/10-containerd-net.conflist  | jq -r ".plugins[] | select(.type == \"bridge\") | .ipam.subnet")
-if [[ -z "${podSubnetAddr}" ]]; then
```

Comment:
unfortunately, no. The way it works today is that cloud-controller manager assigns the node a pod CIDR. Kublet/containerd watch the node, and once it sees the pod CIDR it generates the conflist. The ensure-no-dup.sh script runs in a loop waiting for the conflist to appear so it can read the pod CIDR.

I think longer-term I'd want to see if we can remove this script entirely. From what I saw in K8s PRs from 2017, the bridge was set to promiscuous mode as a workaround for a kernel bug with hairpin mode.